components/workflow-mgt/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/WorkflowImplServiceImpl.java

```diff
@@ -170,6 +291,17 @@ public void deleteHumanTask(WorkflowRequest workflowRequest) throws WorkflowImpl
 
                 }
             }
+            for (WorkflowImplServiceListener workflowListener : workflowListenerList) {
+                try {
+                    workflowListener.doPostDeleteHumanTask(workflowRequest);
+                } catch (WorkflowImplException e) {
+                    throw new WorkflowImplException(
+                            "Error occurred while calling doPostDeleteHumanTask in WorkflowImplServiceListener ," +
+                                    workflowListener.getClass().getName(), e);
+                }
+            }
+
```

Comment:
Listeners must come as the last statement of the method; even after the catch block.